src/conversion/downcasthelpers.js

```diff
@@ -824,8 +826,10 @@ function changeAttribute( attributeCreator ) {
 			 *
 			 * @error conversion-attribute-to-attribute-on-text
 			 */
-			log.warn( 'conversion-attribute-to-attribute-on-text: ' +
-				'Trying to convert text node\'s attribute with attribute-to-attribute converter.' );
+			console.warn( attachLinkToDocumentation(
```

Comment:
This should be `CKEditorError` exception.

Reply:
OTH it doesn't break the editor.

Reply:
But this means that someone used our API in an incorrect way. I think there are 2 ways:  we agreed that using an attribute to attribute converter on text is fine and do not warn about it, or we agreed that it is not fine, and does not allow people to use it (throw `CKEditorError`). It does not make much sense to be in between.

Reply:
I think we should throw now. I will ping @scofalik, if he will know about some cases when it is correct, we will change it later.